src/org/jitsi/impl/neomedia/rtp/StreamRTPManager.java

```diff
@@ -168,14 +171,17 @@ public void putResumableStreamRewriter(
     public ResumableStreamRewriter getResumableStreamRewriter(Long ssrc,
                                                               boolean create)
     {
-        ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-        if (rewriter == null && create)
+        synchronized (ssrcToRewriter)
```

Comment:
Since this commit last week the map is accessed from the RTPConnectorOutputStream thread:
https://github.com/jitsi/libjitsi/commit/87f71421e59531e8a2382fcb398d3688227b0f65